Examples/FireBaseUI/FireBaseUIViewController.swift

```diff
@@ -55,6 +74,17 @@ internal final class FireBaseUIViewController: ViewController {
   var btnReset = Button(frame: Rect(x: 256, y: 324, width: 156, height: 32),
                         title: "Reset Password")
 
+  lazy var fetchUserDocumentButton: Button = {
+    return Button(
```

Comment:
Breaking after the `(` feels really odd to me.  I prefer the coddled form:

```swift
return Button(frame: .init(x: btnReset.frame.maxX, y: btnReset.frame.minY, width: 100, height: 32),
              title: "Get User")
```

Reply:
Personally, I disagree. I think the style provided is nicer since it's easier to adapt as the required signature of `Button` changes. However I think this is another case where the editor/swiftformat should likely be applied in a way where there is just one format to rule them all. Maybe we should setup swiftformat?

In the meantime I'm happy to change this.